diff --git a/Task.swift b/Task-Edited.swift
index adb24c5..cd5db06 100644
--- a/Task.swift
+++ b/Task-Edited.swift
@@ -312,6 +312,9 @@ public func launchTask(taskDescription: TaskDescription, standardOutput: SinkOf<
 							let errorString = (stderrData.length > 0 ? NSString(data: stderrData, encoding: NSUTF8StringEncoding) as String? : nil) ?? ""

 							var description = "A shell task failed with exit code \(terminationStatus)"
+
+							NSLog("errorString: «\(errorString)», description: «\(description)»")
+
 							if !errorString.isEmpty {
 								description += ":\n\(errorString)"
 							}
